applications/nrf5340_audio/src/bluetooth/bt_management/advertising/bt_mgmt_adv.c

```diff
@@ -73,11 +73,56 @@ static void bond_find(const struct bt_bond_info *info, void *user_data)
 	}
 }
 
+static int addr_print(bt_addr_le_t const *const local_addr,
+		      bt_addr_le_t const *const directed_to_addr)
+{
+	char local_addr_str[BT_ADDR_LE_STR_LEN] = {'\0'};
+	char directed_to_addr_str[BT_ADDR_LE_STR_LEN] = {'\0'};
+
+	if (local_addr == NULL) {
+		return -EINVAL;
+	}
+
+	(void)bt_addr_le_to_str(local_addr, local_addr_str, BT_ADDR_LE_STR_LEN);
+	LOG_INF("Local RPA: %s", local_addr_str);
+
+	if (directed_to_addr != NULL) {
+		(void)bt_addr_le_to_str(directed_to_addr, directed_to_addr_str, BT_ADDR_LE_STR_LEN);
+		LOG_INF("Directed to addr: %s.", directed_to_addr_str);
+	}
+
+	return 0;
+}
+
+#if defined(CONFIG_BT_PRIVACY)
+static bool adv_rpa_expired_cb(struct bt_le_ext_adv *adv)
+{
+	int ret;
+	struct bt_le_ext_adv_info ext_adv_info;
+
+	LOG_INF("RPA (Resolvable Private Address) expired.");
+
+	ret = bt_le_ext_adv_get_info(ext_adv, &ext_adv_info);
+	__ASSERT(ret == 0, "bt_le_ext_adv_get_info failed");
```

Comment:
Is there a reason for __ASSERT being used here in stead of ERR_CHK?
Is this related to the discussions about changing from ERR_CHK to __ASSERT?

Reply:
I changed these to ERR_CHK_MSG. We can do the change to asserts in a single focused PR.